inc/templates/search-replace.php

```diff
@@ -22,8 +22,11 @@
 		</tr>
 		<tr>
 			<th><label for="csv"><strong><?php esc_html_e( 'CSV Format Search/Replace:', 'search-and-replace'); ?></strong></label></th>
-			<td><textarea id="csv" cols="62" name="csv" placeholder="search value, replace value (one per line)"><?php $this->get_csv_value() ?></textarea></td>
-		</tr>
+			<td>
+                            <textarea id="csv" cols="62" name="csv" placeholder="<?php esc_html_e("search value, replace value (one per line)" , "search-and-replace")?>" ><?php $this->get_csv_value() ?></textarea>
+                            <p style="font-style: italic"><?php echo _e( "using comma delimited( <b style='font-size:150%'>,</b> ), For example to replace cat with dog: <b>cat,dog</b>" , "search-and-replace" );?></p>
```

Comment:
Thanks @duongcuong96 
Hints, there we should change.
- use single quotes instead of double "
- no formatting inside the php files, we should use a separately css file, that we can minify, change etc.
- we should use escaping, use `esc_html_e()` instead from `_e` and printf to include the html.